src/Http/Controllers/Admin/XlsformSubjectCrudController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace Stats4sd\OdkLink\Http\Controllers\Admin;
+
+use Illuminate\Support\Arr;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use Stats4sd\OdkLink\Models\XlsformSubject;
+use Stats4sd\OdkLink\Imports\XlsformImport;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+use Backpack\CRUD\app\Http\Controllers\Operations\ListOperation;
+use Backpack\CRUD\app\Http\Controllers\Operations\ShowOperation;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;
+use Backpack\CRUD\app\Http\Controllers\Operations\CreateOperation;
+use Backpack\CRUD\app\Http\Controllers\Operations\DeleteOperation;
+use Backpack\CRUD\app\Http\Controllers\Operations\UpdateOperation;
+use Maatwebsite\Excel\Facades\Excel;
+
+/**
+ * Class XlsformCrudController
+ * @package \Stats4SD\OdkLink\Http\Controllers\Admin
+ * @property-read CrudPanel $crud
+ */
+class XlsformSubjectCrudController extends CrudController
+{
+    use ListOperation;
+    use CreateOperation;
+    use UpdateOperation;
+    use DeleteOperation;
+    use ShowOperation;
+
+    public function setup(): void
+    {
+        CRUD::setModel(XlsformSubject::class);
+        CRUD::setRoute(config('backpack.base.route_prefix') . '/xlsform-subject');
+        CRUD::setEntityNameStrings('xlsform subject', 'xlsform subjects');
+    }
+
+    /**
+     * Define what happens when the List operation is loaded.
+     *
+     * @see  https://backpackforlaravel.com/docs/crud-operation-list-entries
+     * @return void
+     */
+    protected function setupListOperation(): void
+    {
+        CRUD::column('name');
+        
+    }
+
+    /**
+     * Define what happens when the Create operation is loaded.
+     *
+     * @see https://backpackforlaravel.com/docs/crud-operation-create
+     * @return void
+     */
+    protected function setupCreateOperation(): void
+    {
+        CRUD::field('name')
+            ->validationRules('required');
+
+    }
```

Comment:
I think we need some information for the user on what this is for:
- In the list operation, I'd suggest a Card [Widget](https://backpackforlaravel.com/docs/5.x/base-widgets#how-to-use) - "before_content" will place it above the table. (See the TPP Case Studies Platform [here](https://github.com/stats4sd/tpp-case-studies/blob/d1c3183a520fcde7030db42d8f9d6aaa5639b097/app/Http/Controllers/Admin/Surveys/FarmSurveyCrudController.php#L60) for an example). 
- In the create operation, I think we need a more informative label or hint.